public class exception {

    void print()
    {
        System.out.println("under exception.");
    }
    
}
